ng2-bootstrap.ts

```diff
@@ -16,3 +31,24 @@ export * from  './components/typeahead';
 export * from  './components/position'
 export * from  './components/common'
 export * from  './components/ng2-bootstrap-config';
+
+export default {
+  directives: [Dropdown, Progress, Tab, TabHeading],
+  components: [
+    Accordion,
+    AccordionPanel,
```

Comment:
Hmm, @jkuri does your script handle `components`?
This is actually not valid angular2 metadata. Essentially all `components` are seen and used in the `directives` metadata AFAIK.